File: src/joblist copy/JobList.js
import React, { useState}from 'react'
import { Grid, Image,Card, Icon,Item, List, Divider, Container, Button, Label  } from 'semantic-ui-react'
import joblistData from './JobListData'

export default function JobList(props) {

    return (
        <>
        {
            joblistData.map((item,index)=>(
                <List size='big' relaxed='very'>
                    <List.Item>
                        <List.Content>
                            <List.Header as='a'>{item.title}</List.Header>
                        </List.Content>
                    </List.Item>
                    <List.Item>
                        <List.Content>
                            <List.Description>{item.company}</List.Description>
                            <List.Description>{item.industry}</List.Description>
                            <List.Description>{item.jobDesc}</List.Description>
                            <List.Description>{item.email}</List.Description>
                            <List.Description>{item.postedOn}</List.Description>
                        </List.Content>
                    </List.Item>
                    <Button color='green'>
                    <Icon name='heart' /> Like
                    </Button>
                    <Button color='orange'>
                        <Icon name="paper plane" />Apply 
                    </Button>
                </List> 
            ))
        }
        </>
    )
}

File: JobApi/JobList.cs
using System;

namespace JobApi
{
    public class JobList
    {
        public int id { get; set; }
        public bool isActive { get; set; }
        public string title { get; set; }
        public string location { get; set; }
        public string industry { get; set; }
        public string picture { get; set; }
        public string company { get; set; }
        public string email { get; set; }
        public string jobDesc { get; set; }
        public DateTime postedOn { get; set; }
    }
}
